src/stores/ui.store.ts

```diff
@@ -583,6 +592,24 @@ end tell`)
           break
         }
 
+        // delete key
+        case 51: {
+          switch (store.focusedWidget) {
+            case FocusableWidget.PROJECT_SELECT: {
+              if(store.projects.length === 0) break
```

Comment:
deletion without confirmation is too dangerous, at least this needs a add a confirmation dialog

Reply:
I don't know how to implement a dialog for react-native for macos I tried dialog but it gives problems, then I don't know if you might like it as ui and how to handle the request/confirmation communication between the store and view widget.

I think it's quite complex if you want to use the dialog.

Take the implemented code as an example if you want to implement this possibility in the future.

Reply:
sorry, this needs a confirmation dialog, the deletion code is easy to do, but the UX needs to work before I implement it